Tests/modules/system_related/sys_test.py

```diff
@@ -315,29 +321,56 @@ def g(a):
             Assert('-> print final\n' in out)
             Assert('(Pdb) ' in out)
         finally:
-            nt.unlink('temp.py')
-
+            os.unlink('temp.py')
 
+def test_call_tracing():
+    def f(i):
+        return i * 2
+    def g():
+        pass
+        
+    # outside of a traceback
+    AreEqual(10, sys.call_tracing(f, (5, )))
+    
+    # inside of a traceback
+    log = []
+    def thandler(frm, evt, pl):
+        if evt == 'call':
+            log.append(frm.f_code.co_name)
+            if log[-1] == 'g':
+                sys.call_tracing(f, (5, ))
+        return thandler
+        
+    sys.settrace(thandler)
+    g()
+    sys.settrace(None)
+    
+    AreEqual(log, ['g', 'f'])
+            
 @skip("win32")
 def test_getrefcount():
     # getrefcount
-    Assert(not hasattr(sys, 'getrefcount'))
+    Assert(hasattr(sys, 'getrefcount'))
 
 @skip("win32 silverlight")
 def test_version():
     import re
-    #E.g., 2.5.0 (IronPython 2.0 Alpha (2.0.0.800) on .NET 2.0.50727.1433)
-    regex = "^\d\.\d\.\d \(IronPython \d\.\d(\.\d)? ((Alpha \d+ )|(Beta \d+ )|(RC \d+ )|())((DEBUG )|()|(\d?))\(\d\.\d\.\d{1,8}\.\d{1,8}\) on \.NET \d(\.\d{1,5}){3}\)$"
-    Assert(re.match(regex, sys.version) != None)
+    # 2.7.5 (IronPython 2.7.5 (2.7.5.0) on .NET 4.0.30319.18444 (32-bit))
+    # 2.7.6a0 (IronPython 2.7.6a0 DEBUG (2.7.6.0) on .NET 4.0.30319.18444 (32-bit))
+    # 2.7.6 (IronPython 2.7.6.3 (2.7.6.3) on .NET 4.0.30319.42000 (32-bit))
+    regex = "^\d\.\d\.\d((RC\d+ )|(a\d+ )|(b\d+ )|( ))\(IronPython \d\.\d(\.\d)?(\.\d)?((RC\d+ )|(a\d+ )|(b\d+ )|( ))?((DEBUG )|()|(\d?))\(\d\.\d\.\d{1,8}\.\d{1,8}\) on ((\.NET)|(Mono)) \d(\.\d{1,5}){3} \(((32)|(64))-bit\)\)$"
+    Assert(re.match(regex, sys.version, re.IGNORECASE) != None)
 
 def test_winver():
     import re
     #E.g., "2.5"
     Assert(re.match("^\d\.\d$", sys.winver) != None)
 
+@skip('cli') # BUG, ps1 is not set in non interactive mode
 def test_ps1():
     Assert(not hasattr(sys, "ps1"))
 
+@skip('cli') # BUG, ps2 is not set in non interactive mode
```

Comment:
Probably, looks like the change comes from https://github.com/IronLanguages/main/commit/b6a245d8cafbdad26feff729c68f66ab11fd23c8